tutorial_src/gue/prog/guelib.py

```diff
@@ -0,0 +1,112 @@
+# This file was automatically generated by SWIG (http://www.swig.org).
```

Comment:
Nie znam się kompletnie na SWIGu.

Reply:
To jest automatycznie wygenerowana biblioteczka do guelib.py
Jednak musi należeć do paczki, gdyż sio nie umie jej wygenerować, umie natomiast wziąć podaną i zlinkować